temba/channels/handlers.py

```diff
@@ -1954,3 +1954,40 @@ def post(self, request, *args, **kwargs):
 
         else:  # pragma: no cover
             return HttpResponse("Not handled, unknown action", status=400)
+
+
+class LINEHandler(View):
```

Comment:
While presenting to the user it's okay to use LINE since that's their brand, but lets stick with Line instead of LINE in code.